### ISSUE TITLE
Triggered Cloudflare Bot Protection

### ISSUE THREAD

**Issue by GTM009** [crunchy-labs/crunchy-cli#386]
Cammand : crunchy login --credentials email: password

Even giving same error for while ripping Content

Output :
<b>:: Triggered Cloudflare bot protection (https://www.crunchyroll.com/index/v2): <!DOCTYPE html><html lang="en-US"><head><title>Just a moment...</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><meta name="robots" content="noindex,nofollow"><meta name="viewport" content="width=device-width,initial-scale=1"><style>*{box-sizing:border-box;margin:0;padding:0}html{line-height:1.15;-webkit-text-size-adjust:100%;color:#313131}button,html{font-family:system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji}@media (prefers-color-scheme:dark){body{background-color:#222;color:#d9d9d9}body a{color:#fff}body a:hover{color:#ee730a;text-decoration:underline}body .lds-ring div{border-color:#999 transparent transparent}body .font-red{color:#b20f03}body .big-button,body .pow-button{background-color:#4693ff;color:#1d1d1d}body #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI0IyMEYwMyIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjQjIwRjAzIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}}body{display:flex;flex-direction:column;min-height:100vh}body.no-js .loading-spinner{visibility:hidden}body.no-js .challenge-running{display:none}body.dark{background-color:#222;color:#d9d9d9}body.dark a{color:#fff}body.dark a:hover{color:#ee730a;text-decoration:underline}body.dark .lds-ring div{border-color:#999 transparent transparent}body.dark .font-red{color:#b20f03}body.dark .big-button,body.dark .pow-button{background-color:#4693ff;color:#1d1d1d}body.dark #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body.dark #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI0IyMEYwMyIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjQjIwRjAzIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}body.light{background-color:transparent;color:#313131}body.light a{color:#0051c3}body.light a:hover{color:#ee730a;text-decoration:underline}body.light .lds-ring div{border-color:#595959 transparent transparent}body.light .font-red{color:#fc574a}body.light .big-button,body.light .pow-button{background-color:#003681;border-color:#003681;color:#fff}body.light #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body.light #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI2ZjNTc0YSIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjZmM1NzRhIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}a{background-color:transparent;color:#0051c3;text-decoration:none;transition:color .15s ease}a:hover{color:#ee730a;text-decoration:underline}.main-content{margin:8rem auto;max-width:60rem;width:100%}.heading-favicon{height:2rem;margin-right:.5rem;width:2rem}@media (width <= 720px){.main-content{margin-top:4rem}.heading-favicon{height:1.5rem;width:1.5rem}}.footer,.main-content{padding-left:1.5rem;padding-right:1.5rem}.main-wrapper{align-items:center;display:flex;flex:1;flex-direction:column}.font-red{color:#b20f03}.spacer{margin:2rem 0}.h1{font-size:2.5rem;font-weight:500;line-height:3.75rem}.h2{font-weight:500}.core-msg,.h2{font-size:1.5rem;line-height:2.25rem}.body-text,.core-msg{font-weight:400}.body-text{font-size:1rem;line-height:1.25rem}@media (width <= 720px){.h1{font-size:1.5rem;line-height:1.75rem}.h2{font-size:1.25rem}.core-msg,.h2{line-height:1.5rem}.core-msg{font-size:1rem}}#challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI2ZjNTc0YSIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjZmM1NzRhIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+);padding-left:34px}#challenge-error-text,#challenge-success-text{background-repeat:no-repeat;background-size:contain}#challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=);padding-left:42px}.text-center{text-align:center}.big-button{border:.063rem solid #0051c3;border-radius:.313rem;font-size:.875rem;line-height:1.313rem;padding:.375rem 1rem;transition-duration:.2s;transition-property:background-color,border-color,color;transition-timing-function:ease}.big-button:hover{cursor:pointer}.captcha-prompt:not(.hidden){display:flex}@media (width <= 720px){.captcha-prompt:not(.hidden){flex-wrap:wrap;justify-content:center}}.pow-button{background-color:#0051c3;color:#fff;margin:2rem 0}.pow-button:hover{background-color:#003681;border-color:#003681;color:#fff}.footer{font-size:.75rem;line-height:1.125rem;margin:0 auto;max-width:60rem;width:100%}.footer-inner{border-top:1px solid #d9d9d9;padding-bottom:1rem;padding-top:1rem}.clearfix:after{clear:both;content:"";display:table}.clearfix .column{float:left;padding-right:1.5rem;width:50%}.diagnostic-wrapper{margin-bottom:.5rem}.footer .ray-id{text-align:center}.footer .ray-id code{font-family:monaco,courier,monospace}.core-msg,.zone-name-title{overflow-wrap:break-word}@media (width <= 720px){.diagnostic-wrapper{display:flex;flex-wrap:wrap;justify-content:center}.clearfix:after{clear:none;content:none;display:initial;text-align:center}.column{padding-bottom:2rem}.clearfix .column{float:none;padding:0;width:auto;word-break:keep-all}.zone-name-title{margin-bottom:1rem}}.loading-spinner{height:76.391px}.lds-ring{display:inline-block;position:relative}.lds-ring,.lds-ring div{height:1.875rem;width:1.875rem}.lds-ring div{animation:lds-ring 1.2s cubic-bezier(.5,0,.5,1) infinite;border:.3rem solid transparent;border-radius:50%;border-top-color:#313131;box-sizing:border-box;display:block;position:absolute}.lds-ring div:first-child{animation-delay:-.45s}.lds-ring div:nth-child(2){animation-delay:-.3s}.lds-ring div:nth-child(3){animation-delay:-.15s}@keyframes lds-ring{0%{transform:rotate(0)}to{transform:rotate(1turn)}}@media screen and (-ms-high-contrast:active),screen and (-ms-high-contrast:none){.main-wrapper,body{display:block}}.rtl .heading-favicon{margin-left:.5rem;margin-right:0}.rtl #challenge-success-text{background-position:100%;padding-left:0;padding-right:42px}.rtl #challenge-error-text{background-position:100%;padding-left:0;padding-right:34px}</style><meta http-equiv="refresh" content="375"></head><body class="no-js"><div class="main-wrapper" role="main"><div class="main-content"><noscript><div id="challenge-error-title"><div class="h2"><span id="challenge-error-text">Enable JavaScript and cookies to continue</span></div></div></noscript></div></div><script>(function(){window._cf_chl_opt={cvId: '3',cZone: "www.crunchyroll.com",cType: 'interactive',cNounce: '29855',cRay: '877e10b78853f494',cHash: '16b8592197bd89c',cUPMDTk: "\/index\/v2?__cf_chl_tk=ZdQRF54kJFnTp.J2uqmYNt7oeeTKyusgv34oH9rwkIw-1713709756-0.0.1.1-1301",cFPWv: 'b',cTTimeMs: '1000',cMTimeMs: '375000',cTplV: 5,cTplB: 'cf',cK: "visitor-time",fa: "\/index\/v2?__cf_chl_f_tk=ZdQRF54kJFnTp.J2uqmYNt7oeeTKyusgv34oH9rwkIw-1713709756-0.0.1.1-1301",md: "Ls8imxyG3iwuJ12R6t15cub1SARZghMf1xJzC4alOZY-1713709756-1.1.1.1-Zw5oArUqHItJ62o9GGxGka24jLYCVO.W9nsAAouzrc0jFQC7MBfXx.7h07wMQHML8NzWhD8_THikDgB7ZYlCBJ7c5qQUcMQ1QMqNAnYASGhT_Vu15zvu8VfWwNJj19Qxq3cxWgeB5yn0NFDCi1ko5Y39m1wxhE3fMwQ3Hd8VP6XkTq6mGlz1RJRuwno9kQBp9MrHGw_kUz6abeZWbQRi8FxJVP4ACrs2qpqq9.brKa5RMZ09aB_dvzoQR_REDpWazU9YLDanyOz6Qk.xx504oiKDNy39nfLk9TNRydS6njwRJNHwtwBGQ0kcmT.w_tbm7CK38nV4pEqVp5ynKL5xbk6M2LKkcqbnMhSsPM8_3.1GaFfWCLlkoFOojt4THhSWF88H1UDlZkyweKqzR3zmWIQ_KTW.3FicHFfMRwA6.tjyBYQzYd4d1tbvZV68MGHd6DXeSesxtBcKlCB23JRYEz9V2.aIJ7TTdxUVrYpe9aWfrnyoHjwGU7A0XCJlys6t1y1hjnzvgEaCvOriJfPL1zM9PH0mD4vWRriUP.cnDG_XODMuWlnM.Oc0UjpWp3t46k.Ci5jLmrOrQzHCaZNFP89doV_PFgZKoFJ08NxPBvSjN4P1IdKYgQNKu002iXg6Eqnk5Tq9KdKqn0FHkoHt7ohaLgdLhr6bsfv.ZSOLRdFTc1OWE0QI0rmW9Q7XfT1Geim8kvz_GmT6UkEs1Q4EBGNj0GaYW1q70I9MJcdE6Qz0wI__YiTdW5CVMyOwT__V__jAcXS6LU3yLCYEbeXKg7JbMuTZ8IdnBvILot.q3JwetretR4gbemr2A2Kk6W3beyx1qTkRYm.Nk4OzVgsaPNpwIYw8_epe.N40AI7rBCDPi7gomiLlzT5jqX27znIbqb1WTa5e0e7bTr7anFm1cbfWzdbLjUzm2C0vz0Yo.pce7LZ_dq2NDAhPByhtSDwHFdMJ98aAoSGC8n8i9BZMvE4VUg7E7aG_MCU3rlzGAaYAfEgjjKwFXLP6K1YVk9wge8e3INKOPvo1reYXfzXPkpjxzcfljsENy9XY_WLUlgFRNcTrWlOHBD.ag0P7cvo7HLuAxOq78tEw3HuBp4MHXvlyxQYUsFPi6UJd7KXLRXkn2fvahlLzg25jDTSqX4MGfXBHFx6wjRMwoX.oNepU5h6PTcBqfxUOd.TOOtVEcBO.DyMlz0r6EkucL6lGa5WkrdS1tmMb.oT44ExPlNEipA",mdrd: "1nZOwzN3qF.P1XWIHbtNNyg3s.syic6RCl2TMugzG5I-1713709756-1.1.1.1-i15CTb3MB95AJY05E6IByAjlUTrL6r9csZEmXTkdSXWUY.eza5zMWVCEbhL4v2bpc87g8JYwHEU4aveiP7BkoN8q7Hd36FbhiazsjhK7jVvShm9dIfNrcrfA79cjK0rRZTujbrxNsKia5i8PhDUmiHcgUtuh264CamdlQCsFwn3Q1hut9.fKGl8e4JRdjovLBkGf6n4C0x.ocrHbxFe1mQNxtkJOdbD6TRjcxnlg45Fgn3K4YM6r3gRhAcpj.L7lSkWjA3qvrmE.lM_qMqjaFP14L_RynrD3w42Imryc6wWSSlPUSua94iPOfhM5O5j4OD5cAs.7HSDoSCvHlFBHaFNmdhAUO5O7.GxCuVYVrVXyc5NcfOO5zl3s3seg_PxIKN45cJV3gz8eNjTpT7FeigtuxuvlgfF4LGIpgEgaezWCW8DqKX8qkgSJLRj_bCo7X2PBEV.or_dBIbJWPOkvIHyiptoXLD2EVLhTplhUM0GG7mNAZkyOAHevqrTeJy15gAcPVkPOFG7os1qtig0RhgG3b8Fmv09hp6zncxTD00W9cQuf4zrA484pLk9W11W6rF8veAYwwI9SxHObHU5bS5Y8aaCSliX3KMrSj__ZTkaq.5Qd8NxVaCfShhzTVc7nXFcRFjAMUKr15XW1CHamR1VcpW5Ip2b1vTTHTjpbqUr6uTkALciqI61WUTtmxhyH82yJPt7_Y0EOHko4eWuiqXrkdg6E5pmNJE1PIKSgmsL5naQAcUsNN7U9fAoRggWkM1.6zzwsUNPMS_.7AmU8z_xh9YGTtH9iOyc1zEEqYJpKcI0hoh4QclzTPgzOtSy5KWkqXVBk39QIL15jlKEZqaxtFD5Dgp8nq9_fcHknq0EO9aR05YB3j6nyGpaaIjScbpEeRKBlxHgrCk3nKKBPoCdtr1cMerxvTdEIoY_AxhqDWPdjiBaGO4kDiHrY14k1BTIvyV6uOIFy5qAEylnMyYqkBftj84QYneEMbyTHrAiSeDF4cyERgfKaMdjRKnPxzW9W_VDio2wVUwopg1oOlZRz0Ef8bbOrLj16gFn3uYv9IGQe3kBT9.MysGXHIwT8.Fz_rn3wD5DF.x1_HrcuKgQc3UY4Wrlj1krK2xWQ9QwWllbPJg4TNogmzmsyBR.JTTD1IuuXts8sGfsfBhQdDmYyb41iJBlHoLUqrm4sUmEhrcbJWjVTCWCZIuw9vaaKpIXXVtWbU22R6jHGAW9crAG_HbZxCCYXRR56sBCvCEEmkrPsZw5AHW.ctoSTlM0._NZ5Yd9DpMvhX2p.BkcjTh0WhKw.sb5vcxvx.tCwMnPGpuE9gvgGjX0V9DxKEyhkD.KOJn_Xg23i6zf4vS6fsL.OTDPvbhf6HZ1iMNY13KuFMWyKhcsbrmEYXkwRv9NpMp5WDuAwSGOSKmfk1Kxc7EqXTiuXUqX8OOyUG7o7a_iA2xncrLnO5bIE5g9RCDqz33QcTtgw0MpAjCVYepHvHL3VtT96CF4NlaSCMHME0CU6vJYw8gbTb0M2A1L0Px9KX4xXE3nE2PN91MpsTcJn4xwmrFy7a9JihgBRo.1_nW4OBqveGmtvUpIDxmOT5jBl1zWHSAg0MA8sJrKD8NAZGRp9MNvAPnGyYGBFoobNdokn9JHw3aypROOQOGQ0T_SYn5rzlMNMDtQyJtXyKm7w4htnq6.PsvjVvLNCv4WOcH58cg_LMYV94V_fG57fmyQGDOdicHAIyrBy0vfYv14Z75OPPze8ZsTkbOUP25xc3j1BLG.COmi.YcEuKAc29C8EcEJojuN.OVjkvBo9l91rtEIFs77vsA7M6kvHQtC17p86TtvO4yMaVSW7j1bhwBa154BNo19w1IJeNL6I4MOna4W.2raJ3IFEo8k62YALzdPQJlj6ri8tU92_BbH0SZMMU_qiGinH.zYNUkOQInVBjzjhYglyURhhg.YORBGMijuG85n1lFuDdc0Ek28XtU5KVbBpIcqDGRtnimVOUoyNyTHVJg9210naYj9rjxh_CyWuL3YBMbznLCACAbQMoW.rfB4BSu7NSfZRC29o.gdS6scBrsQfGCfFfsMnZwSIYm1pzTOTzUDQbSYzyErfzDAC8t3m9F5t.N90m_2ONa_ha6dM30M.D1MV.TWmvIB1WAFpK0CagwkoXYiDUTsJk05t29xqFvy0RAi8St.0QPmpbf84HTYLr_fhMCBPszpWjgDxj6N6307_bS384KFqtPZJODcbSmpCx4o2wrSxBkdNO6TpUpqHPe.7rEoKOEC7MQ87tz1A4H8rgqC_IT9aMJxX9mHUadCrf5w48FawrjsCRYoWDNB4UBlW.OW1ItwxkBotDgbiYNBU8UrNc2C4.oCNnliOAoWJip1TLMDmxPca02jEjr_8Q._kgUqpp.dNe27dOs4dmiW2Atd6ypBW4KyNwlgwCTD2MTKp.CJ5erHu7k8Y59O106a_2uyuwZEls.lkAr3KvQlu7kdZpbEmPHp7jL2ETqLvddgzckN6cw_nr2jf.fo0.leF6Qv2d6lFNnfc04eYWYQ.JHku8vDcjkHFmQwkZNFMBZGHTLt9yIRXKPT5AOzemZZCeSoUlnn0XnsF2N4pjYC0snP8d4CnTnY6ZQRck9GLvvQR_4MEFUKXR0f0D6gnTnXzMJEWxgYqXeKyXbL40p.q8Nzg.3vecJ4V2nRwVDrbP27sW1YiICL60PrmS.kjLC_GHW1V6BJYV0IC1Jaj3sIOhphqJPpNEG24KJcpY0bXL5kC.Y0k8rwPQwXXRYIgfpflsNY8r9Vo9vibeLCuZC_XmP4tr.u1ZXubaQerVfEnF9oq5mdsfKgB6.BV4G8vRehOmy9l1PTOIMUc.pDvJC3JdV6ir_LJO_liXplaJUFPnpJAkFlcZnnTV.Hpx2jZU0sdMI9DtgHUxykxjbzah.UdJ.ghxNviiRll6vdvayy6Hhu4aHlw.kdId482AuEmDotqMkj7yYsO1UkQR_PbFgG5ZJx1sNYR_BF_14LamTDqf.ve803FQ06rR6cifb7Q0fxlAxalWcssC0a6RkYd.ZBjqhvMd4bsZmdytqZAB9nILpvpJ.W7TgWUQI_mjbNGp5k1m7WqWGTvRqqFo1Pg2wNu3RMUJjvHW3bL4Ko8CmzmZNdf26.ovCTsXr5vFjF6zKFz56dDZ3OiDbejhM.P0kPR2eERb63.jbNVJIKBbLTfl8OCLXn3XJSWgwZasFmhJ2bd_n3DLHAMmYsgXtmdF.qeb10Ig.I7fOZOgpvXaoZtr3aq7NRlOIxm51_7pMap_o75YEbWVkWJowLwvrJT2d8tk7wsibYZuvEc574ta5EMpEX.iA8vsYctS8XnXgm_AUf27t1kNvcpg7u7cTGtz4KCRfENLYy3YwV30QmCzgAZzEKtTQ2Ilw3Ej5AxH4mexG_jwSg2SFOnea_rldA4T26vT3tpehGARuW850IZM360gbdj64PGWt7tnQRw0xq5AHE87UqvJQln6ST3l0xOEy05Hvx0Ns_ogZOM",cRq: {ru: 'aHR0cHM6Ly93d3cuY3J1bmNoeXJvbGwuY29tL2luZGV4L3Yy',ra: 'TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NCkgQXBwbGVXZWJLaXQvNTM3LjM2IChLSFRNTCwgbGlrZSBHZWNrbykgQ2hyb21lLzExNC4wLjAuMCBTYWZhcmkvNTM3LjM2IEVkZy8xMTQuMC4xODIzLjY3YQ==',rm: 'R0VU',d: 'OWVbWiidIx7pZ7K4Po6rQhcUetK0G7YaMJbWH/KPCHcQUi9S8srK+uMuBPxg11oezqvQ9BdA+rtqT2rEtx3Mx7kYT8Tz1huibyhkGPUK9iqseChsNP10DZvCMVGEj7gjCqbYl2V2uFkc8LeewusgF7yrO3lam6dLjnkf8Jf79F5/oJujBRgyDts4ewWZuv+qS0FVtQG4LnEV5S0xBAH3GB/GJNJkCHwkZ/V3HP2JYC+QAwJVI5qNDi+aI3seA3wgxshmupdlpT0lvu7TOGMZKi9H6MJekTXs2UER9xQqjXOKg9VawCfLWM7g0J82mGxeORINeqoxNuUb+AofncF7TzhNZJDH0MOuMGMDvQcT0dBJMdpm60YfX36d/J26DnebMcu0yCiVb/mO083lWTFZTIBMtIiW5TYM9TvFiUpNxtQ+/E3v/wWOqMyOpJUiX6TRsG4+6xCatZpSp/jBnZgMHoCKVZTBR7gyoHHHiMSVDg90i+lWRACOpvxhwQDzTKMeyrUKqdbxng3t8OIvUv8fFMMJ5qMo5+YTlvxEBA9nfEKp79WcZDdImKRdOea+FqN1/4/ygkU6vngWMZQ9/xz6CQ==',t: 'MTcxMzcwOTc1Ni4wOTYwMDA=',cT: Math.floor(Date.now() / 1000),m: 'mBr3LN87qB/blldEK0aDrXCVt92FZFEgZEgZZZylv6g=',i1: 'ay1iaia/iDRP8fNQPZupwA==',i2: 'ZV0gBfn1ljFQ4F2eutWROA==',zh: 'qQz4tblgEgtOKuLwUaA7NrEUXyzFHvjAH4jkpQCn35I=',uh: 'khEF8jUex0IHaWvQrTOSJ7wstwSk8MX26uyppCHwbfo=',hh: 'X/ui56Hpfhv5vN0FUDU58Tk8kOdH4hwp+B7yjRL1zVA=',}};var cpo = document.createElement('script');cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=877e10b78853f494';window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;if (window.history && window.history.replaceState) {var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;history.replaceState(null, null, "\/index\/v2?__cf_chl_rt_tk=ZdQRF54kJFnTp.J2uqmYNt7oeeTKyusgv34oH9rwkIw-1713709756-0.0.1.1-1301" + window._cf_chl_opt.cOgUHash);cpo.onload = function() {history.replaceState(null, null, ogU);}}document.getElementsByTagName('head')[0].appendChild(cpo);}());</script></body></html></b>

**Comment by westhecool** [comment]
Are you using a VPN?

**Comment by GTM009** [comment]
> Are you using a VPN?

No Its VPS (From The Starting I Use Crunchy On VPS)

**Comment by westhecool** [comment]
> > Are you using a VPN?
> 
> No Its VPS (From The Starting I Use Crunchy On VPS)
> 
> Edit : sometimes its working but most of the time i am getting this error

Cloudflare probably marked the IP of your server as suspicious. You can try using a VPN or proxy.

**Comment by GTM009** [comment]
Proxy will be costly, need a way to Bypass it because sometimes its working
I even ripped whole season

**Comment by GTM009** [comment]
> > > Are you using a VPN?
> > 
> > 
> > No Its VPS (From The Starting I Use Crunchy On VPS)
> > Edit : sometimes its working but most of the time i am getting this error
> 
> Cloudflare probably marked the IP of your server as suspicious. You can try using a VPN or proxy.

Proxy will be costly, need a way to Bypass it because sometimes its working
I even ripped whole season

**Comment by GTM009** [comment]
Closing this as i got solution for this
I just Adding User-Agent in every cmd to avoid this issue 
Thanks all

**Comment by GTM009** [comment]
User-Agent didn't worked

**Comment by Kartatz** [comment]
I don't see how this is a crunchy-cli issue.

Cloudflare is blocking you due to suspicious network traffic. There is nothing crunchy-cli can do to prevent that.

**Comment by bytedream** [comment]
This is a duplicate of #104.
I already did the best I can to construct the requests so that Cloudflare doesn't get triggered but it cannot be 100% avoided. I have the same setup as you, crunchy-cli on a VPS, and get blocked by Cloudflare too. The only way to prevent this from happening it using a VPN on the VPS an tunnel all traffic through it.
